pypeit/core/flux_calib.py

```diff
@@ -948,17 +958,17 @@ def fit_zeropoint(wave, Nlam_star, Nlam_ivar_star, gpm_star, std_dict,
         sensfactor = Nlam_to_Flam(wave, zeropoint_fit)
         plt.plot(wave[zeropoint_fit_gpm], flux_true[zeropoint_fit_gpm], color='k',lw=2, label='Reference Star')
         plt.plot(wave[zeropoint_fit_gpm], Nlam_star[zeropoint_fit_gpm]*sensfactor[zeropoint_fit_gpm], color='r', label='Fluxed Observed Star')
-        plt.xlabel(r'Wavelength [$\AA$]')
-        plt.ylabel('Flux [erg/s/cm2/Ang.]')
+        plt.xlabel('Wavelength (Angstroms)')
+        plt.ylabel(r'Flux (erg/s/cm$^2$/$\AA$)')
         plt.legend(fancybox=True, shadow=True)
         plt.show()
 
     return zeropoint_data, zeropoint_data_gpm, zeropoint_fit, zeropoint_fit_gpm
 
 
 def get_mask(wave_star, flux_star, ivar_star, mask_star, 
-             mask_abs_lines=True, mask_telluric=True, balm_mask_wid=10., 
-             trans_thresh=0.9):
+             mask_hydrogen_lines=True, mask_helium_lines=False,
```

Comment:
Should we have a helium mask width parameter or is it just using the Balmer_mask_width. If so, maybe document that.

Reply:
Added documentation about the `HeII` mask width being set to 0.5 x the hydrogen width.